tutoraspects/templates/aspects/apps/superset/pythonpath/create_assets.py

```diff
@@ -64,32 +70,47 @@ def create_assets():
             path = os.path.join(root, file)
             with open(path, "r") as file:
                 asset = yaml.safe_load(file)
-                process_asset(asset, roles)
+                process_asset(asset, roles, translated_asset_uuids)
 
-    with open(ASSETS_FILE_PATH, "r") as file:
+    # Get extra assets
+    with open(os.path.join(PYTHONPATH,"assets.yaml"), "r") as file:
         extra_assets = yaml.safe_load_all(file)
 
         if extra_assets:
             # For each asset, create a file in the right folder
             for asset in extra_assets:
-                process_asset(asset, roles)
+                process_asset(asset, roles, translated_asset_uuids)
+
+
+    for uuid in translated_asset_uuids:
+        translated_asset_uuids[uuid] = list(translated_asset_uuids[uuid])
+
+    # Write parent UUID & translated child UUIDs to yaml file
+    path = os.path.join(PYTHONPATH,"translated_asset_mapping.yaml")
+    with open(path, "w") as file:
```

Comment:
Looks like this is making k8s sad since you can't write files there. Could you keep this is memory and just pass it to `delete_assets`, or am I missing something?

Reply:
yup good call, ill do that